src/BundleAnalyzerPlugin.js

```diff
@@ -23,7 +23,18 @@ class BundleAnalyzerPlugin {
       logLevel: 'info',
       // deprecated
       startAnalyzer: true,
+
+      // JSON specific options block
+      failBuild: true,
       initialLoadingResources: [],
+      maxInitialLoadingSizeSingle: 100,
+      maxInitialLoadingSizeBundle: 500,
+      chunksLoadingResources: [],
+      maxLazyLoadingSizeSingle: 100,
+      failOnBudgetError: true,
+      // Lazy Bundle size check disabled initially
+      maxLazyLoadingSizeBundle: Number.MAX_SAFE_INTEGER,
```

Comment:
Don't you think it's better to set it 100, for example?
To have at least some default value, what do you think?

Reply:
Set to 1000, 100 would give a warning, for lazy components it's not needed now. I made this option in case we need it in future.